std_msgs/Header header
CircleObstacle[] circles